### PR TITLE
Release notes for Builds 1.2

### PR DESCRIPTION
**Pull request by eromanova97**
Issue: 

Link to docs preview: https://85324--ocpdocs-pr.netlify.app/openshift-builds/latest/release_notes/ob-release-notes.html#ob-release-notes-1-2_ob-release-notes

**Additional information:**

### CODE REVIEW

File: modules/ob-release-notes-1-2.adoc [hunk 1]
```diff
@@ -5,20 +5,69 @@
...
 
 The following section highlights known issues in {builds-shortname} 1.2.
 
+* The default build strategies fail on clusters with ARM, IBM Power, or IBM Z worker node CPU architectures. 
++
+As a workaround, perform the following steps:
+
+.. Download the following multi variant of the default `ClusterBuildStrategy` resources to your cluster:
+** `buildah-multi`: link:https://issues.redhat.com/secure/attachment/13306657/13306657_buildah-multi.yaml[buildah-multi.yaml]
+** `source-to-image-multi`: link:https://issues.redhat.com/secure/attachment/13306658/13306658_source_to_image_multi.yaml[source_to_image_multi.yaml]
```
Comment (avinal) [inline modules/ob-release-notes-1-2.adoc:56]: The links are correct and publicly accessible, but please check if it is the preferred way to provide the files. The docs team will have a guideline for this.

**Review by avinal — APPROVED** [review]
One suggestion. Looks good to me.

File: modules/ob-release-notes-1-2.adoc (hunk 1 above)
Comment (eromanova97) [reply modules/ob-release-notes-1-2.adoc:56]: Hello, I am not aware of any rules regarding this. i will run this through peer review and see their opinion. If they do not know, I will find the answer before merging. Thanks!

File: modules/ob-compatibility-support-matrix.adoc [hunk 2]
```diff
@@ -23,8 +23,8 @@ The link:https://access.redhat.com/support/offerings/techpreview[Technology Prev
 
 | Operator | Builds (Shipwright) | CLI | | |
 
-|1.1 | 0.13.0 (GA) | 0.13.0 (GA) | 1.13, 1.14, and 1.15        | 4.12, 4.13, 4.14, 4.15, and 4.16         | GA
-|1.0 | 0.12.0 (GA) | 0.12.0 (GA) | 1.12, 1.13, 1.14, and 1.15 | 4.12, 4.13, 4.14, 4.15, and 4.16         | GA
-|1.2 |  |  |        |          | GA
+|1.2 | 0.13.0 (GA) | 0.13.0 (GA) | 1.13, 1.14, 1.15, and 1.16 | 4.12, 4.13, 4.14, 4.15, 4.16, and 4.17 | GA
```
Comment (adambkaplan) [inline modules/ob-compatibility-support-matrix.adoc:26]: Maintenance support for OpenShift Pipelines 1.13 ended on October 9 - we should not support it.

```suggestion
|1.2 | 0.13.0 (GA) | 0.13.0 (GA) | 1.14, 1.15, and 1.16 | 4.12, 4.13, 4.14, 4.15, 4.16, and 4.17 | GA
```

File: modules/ob-release-notes-1-2.adoc [hunk 3]
```diff
@@ -5,20 +5,69 @@
 [id="ob-release-notes-1-2_{context}"]
 = Release notes for {builds-shortname} General Availability 1.2
 
-{builds-shortname} General Availability (GA) 1.2 is now available on {ocp-product-title} <version>.
+{builds-shortname} General Availability (GA) 1.2 is now available on {ocp-product-title} 4.17.
```
Comment (adambkaplan) [inline modules/ob-release-notes-1-2.adoc:8]: Shouldn't we indicate this is available on all supported versions of OpenShift?

```suggestion
{builds-shortname} General Availability (GA) 1.2 is now available on {ocp-product-title} 4.12-4.17.
```

File: modules/ob-release-notes-1-2.adoc [hunk 4]
```diff
@@ -5,20 +5,69 @@
...
 == New features
 
-The following sections highlight what is new in {builds-shortname} 1.2.
+The following section highlights what is new in {builds-shortname} 1.2.
 
 [id="builds-new-features-1-1_{context}"]
-=== Builds
+=== {builds-shortname}
 
+* With this update, you can use the {builds-title} on the {ocp-product-title} cluster that runs on ARM, IBM Power, and IBM Z worker hardware. The components to build images with Shipwright and Shared Resource CSI Driver are available.
```
Comment (adambkaplan) [inline modules/ob-release-notes-1-2.adoc:18]: Indicate this is **Technology Preview**?

```suggestion

* Technology Preview: With this update, you can use the {builds-title} on the {ocp-product-title} cluster that runs on ARM, IBM Power, and IBM Z worker hardware. The components to build images with Shipwright and Shared Resource CSI Driver are available.
```

File: modules/ob-release-notes-1-2.adoc [hunk 5]
```diff
@@ -5,20 +5,69 @@
...
+
+* With this update, you can deploy {builds-shortname} on the clusters that are configured to run behind a network proxy.
+
+* With this update, you can deploy {builds-shortname} on the clusters that run in a disconnected network environment.
+
+* With this update, you can use the `buildah` and `source-to-image` build strategies to mount a Red{nbsp}Hat subscription entitlement key from any supported volume source. This is useful when you need builds to install Red{nbsp}Hat subscription content by using `yum` or `dnf` commands.
```
Comment (adambkaplan) [inline modules/ob-release-notes-1-2.adoc:24]: Issue with `{nbsp}`?

```suggestion
* With this update, you can use the `buildah` and `source-to-image` build strategies to mount a Red Hat subscription entitlement key from any supported volume source. This is useful when you need builds to install Red Hat subscription content by using `yum` or `dnf` commands.
```

File: modules/ob-release-notes-1-2.adoc [hunk 6]
```diff
@@ -5,20 +5,69 @@
...
+
+.. Run the following command to update your `Build` resource to use the corresponding build strategy:
++
+.Buildah
+[source,terminal]
+----
+$ oc apply -f buildah-multi.yaml
+----
++
+.S2i
```
Comment (adambkaplan) [inline modules/ob-release-notes-1-2.adoc:66]: Use full name?

```suggestion
.Source-to-image
```

**Review by adambkaplan — COMMENTED** [review]
A few "nits" - otherwise looks good.

File: modules/ob-release-notes-1-2.adoc (hunk 3 above)
Comment (eromanova97) [reply modules/ob-release-notes-1-2.adoc:8]: That would be definitely better :+1: I will add that change.

Comment (eromanova97) [reply modules/ob-release-notes-1-2.adoc:8]: I will unify it with other outer loop by saying "4.12 and later versions"

File: modules/ob-release-notes-1-2.adoc (hunk 4 above)
Comment (eromanova97) [reply modules/ob-release-notes-1-2.adoc:18]: Yes, if it is a tech preview, it should definitely be indicated :+1:

File: modules/ob-release-notes-1-2.adoc (hunk 6 above)
Comment (eromanova97) [reply modules/ob-release-notes-1-2.adoc:66]: Yes, I can change that, no problem!

File: modules/ob-release-notes-1-2.adoc [hunk 7]
```diff
@@ -5,20 +5,73 @@
...
 == New features
 
-The following sections highlight what is new in {builds-shortname} 1.2.
+The following section highlights what is new in {builds-shortname} 1.2.
 
 [id="builds-new-features-1-1_{context}"]
-=== Builds
+=== {builds-shortname}
 
+* With this update, you can use the {builds-title} on the {ocp-product-title} cluster that runs on ARM, IBM Power, and IBM Z worker hardware. The components to build images with Shipwright and Shared Resource CSI Driver are available.
```
Comment (mburke5678) [inline modules/ob-release-notes-1-2.adoc:18]: _worker hardware_ is an odd term to me. 

```suggestion
* With this update, you can use the {builds-title} on an {ocp-product-title} cluster that runs on the ARM, IBM Power, or IBM Z platforms. The components to build images with Shipwright and Shared Resource CSI Driver are available.
```

File: modules/ob-release-notes-1-2.adoc (hunk 1 above)
Comment (mburke5678) [reply modules/ob-release-notes-1-2.adoc:60]: I am OK with the links if the downloaded files have been tested and passed by QE.

Comment (mburke5678) [reply modules/ob-release-notes-1-2.adoc:60]: Is _multi variant_ a known term? I am not seeing too much relevant by using web searches. There are some uses of multi-variant as an adjective (multi-variant testing, for example).

File: modules/ob-release-notes-1-2.adoc [hunk 8]
```diff
@@ -5,20 +5,73 @@
...
 
+* The default build strategies fail on clusters with ARM, IBM Power, or IBM Z worker node CPU architectures. 
++
+As a workaround, perform the following steps:
+
+.. Download the following multi variant of the default `ClusterBuildStrategy` resources to your cluster:
+** `buildah-multi`: link:https://issues.redhat.com/secure/attachment/13306657/13306657_buildah-multi.yaml[buildah-multi.yaml]
+** `source-to-image-multi`: link:https://issues.redhat.com/secure/attachment/13306658/13306658_source_to_image_multi.yaml[source_to_image_multi.yaml]
+
+.. Run the following command to update your `Build` resource to use the corresponding build strategy:
```
Comment (mburke5678) [inline modules/ob-release-notes-1-2.adoc:62]:
```suggestion
.. Run one of the following commands to update your `Build` resource to use the corresponding build strategy:
````

Comment (mburke5678) [reply modules/ob-release-notes-1-2.adoc:62]: Or better, this structure is far more prevalent in the OCP docs:

```suggestion
.. Update your `Build` resource to use the corresponding build strategy by running one of the following commands:
```

File: modules/ob-release-notes-1-2.adoc [hunk 9]
```diff
@@ -5,20 +5,73 @@
...
+
+* With this update, you can download the binaries for `shp` CLI from the *Developer* perspective of the {ocp-product-title} web console. It is available for the following operating systems and architectures:
+
+[options="header"]
+|===
+| Operating system | Architecture
+| Linux  | amd64
+| Windows | amd64
+| Darwin | amd64 (macOS Intel)
+|===
```
Comment (mburke5678) [inline modules/ob-release-notes-1-2.adoc:40]: Add a `+` before the table to better align with the bullet.
```suggestion
* With this update, you can download the binaries for `shp` CLI from the *Developer* perspective of the {ocp-product-title} web console. It is available for the following operating systems and architectures:
+
[options="header"]
|===
| Operating system | Architecture
| Linux  | amd64
| Windows | amd64
| Darwin | amd64 (macOS Intel)
|===
```

**Comment by mburke5678** [comment]
@eromanova97 A few comments. Otherwise LGTM

File: modules/ob-release-notes-1-2.adoc (hunk 1 above)
Comment (eromanova97) [reply modules/ob-release-notes-1-2.adoc:61]: Sorry I missed your comments! Not sure about how well the term is known, I will keep it for not to not postpone the release,but I will add the internal website note, thanks!

**Pull request merged**